distribution/osbuild-composer.service

```diff
@@ -15,6 +15,7 @@ Restart=on-failure
 
 # systemd >= 240 sets this, but osbuild-composer runs on earlier versions
 Environment="CACHE_DIRECTORY=/var/cache/osbuild-composer"
+Environment="STATE_DIRECTORY=/var/lib/osbuild-composer"
```

Comment:
Heh, I almost wanted to say that setting `Environment` doesn't work, but of course it does: https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Environment=